src/Core/src/Handlers/MenuBarItem/MenuBarItemHandler.cs

```diff
@@ -22,6 +22,7 @@ public partial class MenuBarItemHandler : IMenuBarItemHandler
 		{
 #if WINDOWS
 			[nameof(IMenuBarItem.Text)] = MapText,
+#elif MACCATALYST || IOS || WINDOWS
```

Comment:
Not sure this is correct. If windows then text else if windows enabled. I think because windows is in both, it needs to be separated?

Reply:
If Windows and nothing else, MapText.
If Windows, iOS, or Catalyst, MapIsEnabled.

Would you prefer this?
```
#if WINDOWS
			[nameof(IMenuBarItem.Text)] = MapText,
            [nameof(IMenuBarItem.IsEnabled)] = MapIsEnabled,
#elif MACCATALYST || IOS 
			[nameof(IMenuBarItem.IsEnabled)] = MapIsEnabled,
#endif
```